classes/Provider/Gateways/RequestCleaner.php

```diff
@@ -32,8 +32,11 @@ private function clean(array $data)
             if (is_array($value)) {
                 $sanitized[$key] = $this->clean($value);
             } else {
-                $sanitized[$key] = $this->purifier->purify($value);
-                ;
+                $sanitized[$key] = preg_replace(
+                    ['/&amp;/', '/&lt;\b/', '/\b&gt;/'],
```

Comment:
What cases does the `\b` usage cover? And why do we have it only on `<` and `>`, but not `&`?